src/webview/static/base.css

```diff
@@ -569,6 +569,14 @@ hr {
   border-top: 1px solid hsla(0, 0%, 50%, 0.5);
 }
 
+h1, h2, h3, h4, h5, h6 {
+  font-size: 100%;
+  font-weight: 600;
+  margin-top: 0;
+  margin-bottom: 0.5rem;
```

Comment:
Margin should be in `px`:
https://github.com/zulip/zulip-mobile/blob/master/docs/style.md#px-rem-no-em

Reply:
Done.